bee-api/bee-rest-api/Cargo.toml

```diff
@@ -29,6 +29,7 @@ hex = { version = "0.4" }
 iota-crypto = { git = "https://github.com/iotaledger/crypto.rs", rev = "c3bf565eba62d0b81144174c2ff917bfde282e49", features = ["blake2b"], optional = true }
 log = { version = "0.4", optional = true }
 num_cpus = { version = "1.13", optional = true }
+libp2p = { version = "0.35", default-features = false }
```

Comment:
Update `libp2p` version to "0.36".

Reply:
I guess the same would apply to port initialization in `bee-node/Cargo.toml`?

Reply:
yes :+1: